core/services/ocr2/plugins/functions/config/config_types.proto

```diff
@@ -9,11 +9,26 @@ enum AggregationMethod {
     AGGREGATION_MEDIAN = 1;
 }
 
+message OracleIDtoKeyShareIndex {
+  uint32 oracle_id = 1;
+  uint32 key_share_index = 2;
+}
+
+message ThresholdReportingPluginConfig {
```

Comment:
You should include the one from tdh2, not copy-paste.

Reply:
According to a quick search, types from a .proto cannot be imported from an external Go package.

Reply:
You import from another .proto file, not from a Go package.